packages/core/src/index.ts

```diff
@@ -65,3 +66,13 @@ export {
 export type Actor = ActorType;
 
 export * from './types';
+
+// TODO: decide from where those should be exported
```

Comment:
Those had to be exposed because they were used by other packages. By switching to preconstruct we are effectively hiding all internal modules from the outside world, so those couldn't be accessed anymore by reaching into `xstate/lib`.